blazor/ai-assistview/methods.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title: Methods in Blazor AI AssistView Component | Syncfusion
+description: Checkout and learn here all about Methods with Syncfusion Blazor AI AssistView component in Blazor Server App and Blazor WebAssembly App.
+platform: Blazor
+control: AI AssistView
+documentation: ug
+---
+
+# Methods in Blazor AI AssistView component
+
+## Executing prompt
```

Comment:
missed **Adding prompt response** section add it @SenthilarasuBalu

Reply:
In blazor only ExecutePromptAsync is present.